flink-yarn/src/test/java/org/apache/flink/yarn/YarnClusterDescriptorTest.java

```diff
@@ -830,4 +832,42 @@ private YarnClusterDescriptor createYarnClusterDescriptor(Configuration configur
                 .setYarnClusterInformationRetriever(() -> YARN_MAX_VCORES)
                 .build();
     }
+
+    @Test
+    public void testGenerateApplicationMasterEnv() throws IOException {
+        final Configuration flinkConfig = new Configuration();
+        final File flinkHomeDir = temporaryFolder.newFolder();
+        final String fakeLocalFlinkJar = "./lib/flink_dist.jar";
+        final String fakeClassPath = fakeLocalFlinkJar + ":./usrlib/user.jar";
+        final ApplicationId appId = ApplicationId.newInstance(0, 0);
+        try (final YarnClusterDescriptor yarnClusterDescriptor =
+                createYarnClusterDescriptor(flinkConfig)) {
+            final YarnApplicationFileUploader yarnApplicationFileUploader =
+                    YarnApplicationFileUploader.from(
+                            FileSystem.get(new YarnConfiguration()),
+                            new Path(flinkHomeDir.getPath()),
+                            new ArrayList<>(),
+                            appId,
+                            DFSConfigKeys.DFS_REPLICATION_DEFAULT);
+            final Map<String, String> masterEnv =
+                    yarnClusterDescriptor.generateApplicationMasterEnv(
+                            yarnApplicationFileUploader,
+                            fakeClassPath,
+                            fakeLocalFlinkJar,
+                            appId.toString());
+
+            Assert.assertEquals("./lib", masterEnv.get(ConfigConstants.ENV_FLINK_LIB_DIR));
```

Comment:
```suggestion
            assertEquals("./lib", masterEnv.get(ConfigConstants.ENV_FLINK_LIB_DIR));
```

Reply:
Fixed in 0b12e096e97eff0fddc7a2e777f4cd40be5a4df3